modules/aleph/src/Aleph/Entity/AlephHoldGroup.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\aleph\Aleph\Entity;
+
+/**
+ * Class AlephHoldGroup.
+ *
+ * Entity for hold groups.
```

Comment:
From a documentation perspective the DocBlocks in this class adds close to nothing.